stingray/tests/test_crossspectrum.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import division
+import numpy as np
+from nose.tools import raises
```

Comment:
I think we agreed to switch from `nosetests` to `py.test` (to comply with `astropy`). 
There's an equivalent to the `raises` decorator in `py.test`: 
` with pytest.raises(AssertionError):
            assert Powerspectrum(nonsense_data)`

See also PR #89, which I need to fix and actually merge.